test/extended/jobs/jobs.go

```diff
@@ -9,7 +9,7 @@ import (
 	exeutil "github.com/openshift/origin/test/extended/util"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/kubernetes/pkg/apis/batch"
+	batchv1 "k8s.io/kubernetes/pkg/apis/batch/v1"
```

Comment:
@soltysh did we switch storage version to batch/v1 already by 1.5 and run a migration prior to 1.6 to ensure we don't have any extensions/v1beta1 resources in etcd any more?